docs/releases/v8_4.md

```diff
@@ -0,0 +1,48 @@
+
+    ![DietPi-Software screenshot](../assets/images/dietpi-software.jpg){: width="480" height="218" loading="lazy"}
+
+- **DietPi-Software** | [**Box86**](../../software/gaming/#box86) :octicons-arrow-right-16: On ASUS Tinker Board, compiling is now done with an optimised build target flag. You can update Box86 by reinstalling it: dietpi-software reinstall 62
+- **DietPi-Software** | [**Box64**](../../software/gaming/#box64) :octicons-arrow-right-16: On Odroid N2, compiling is now done with an optimised build target flag. You can update Box64 by reinstalling it: dietpi-software reinstall 197
+- **DietPi-Software** | [**VirtualHere**](../../software/remote_desktop/#virtualhere) :octicons-arrow-right-16: On new installs and reinstalls, VirtualHere is now installed to `/opt/virtualhere` to align with most other software options. Logging is now done to system log, viewable via `journalctl -u virtualhere`, instead of using the `/var/log/virtualhere`.log plain text file. By default, the system hostname is now used as VirtualHere server name, instead of the hardcoded "DietPi". An existing config file is never touched on reinstalls.
+- **DietPi-Software** | [**ruTorrent**](../../software/bittorrent/#rtorrent) :octicons-arrow-right-16: The ruTorrent web interface is now reinstalled cleanly, i.e. configs, 3rd party plugins and themes are preserved but all other files are removed before new ruTorrent files are installed. Many thanks to @shanew1694 for reporting a related issue with remaining obsolete files: <https://github.com/MichaIng/DietPi/issues/5161#issuecomment-1013286248>
+
+### Bug Fixes
+
+- **Network** :octicons-arrow-right-16: Resolved an issue on some Armbian based systems where the network interface naming changed unintentionally after a kernel upgrade: <https://dietpi.com/phpbb/viewtopic.php?t=10229>
+- [**DietPi-Config**](../../dietpi_tools/#dietpi-configuration) :octicons-arrow-right-16: Resolved an issue on RPi where disabling and re-enabling Bluetooth failed, since the `hciuart.service` cannot be restarted until reboot after it was stopped once. In the same turn, this also resolved an issue in DietPi-PREP where the service is still running after package removal, while trying to disabling it fails as of the missing service file. Many thanks to @Mausy5043 for reporting this issue: <https://github.com/MichaIng/DietPi/issues/5435>
+- **DietPi-Software** | [**Chromium**](../../software/desktop/#chromium) :octicons-arrow-right-16: Resolved an issue on Raspberry Pi Buster systems where the installation failed because of a syntax error. Many thanks to @bbmak for reporting this issue: <https://dietpi.com/phpbb/viewtopic.php?t=10258>
```

Comment:
```suggestion
- **DietPi-Software** | [**Chromium**](../../software/desktop/#chromium) :octicons-arrow-right-16: Resolved an issue on Raspberry Pi Buster systems where the installation failed because of a syntax error. Many thanks to [phpBB:bbmak](https://dietpi.com/phpbb/memberlist.php?username=bbmak){: class="nospellcheck"} for reporting this issue: <https://dietpi.com/phpbb/viewtopic.php?t=10258>
```